### PR TITLE
[stable13] lower log level for quota manipulation cases

### PR DESCRIPTION
**Pull request by blizzz**
backport of #10034

### CODE REVIEW

**Review by MorrisJobke — APPROVED** [review]

File: apps/user_ldap/lib/User/User.php [hunk 1]
```diff
@@ -498,44 +498,39 @@ public function updateQuota($valueFromLDAP = null) {
...
-					} else {
-						$this->log->log('not suitable LDAP quota found for user ' . $this->uid . ': [' . $aQuota[0] . ']', \OCP\Util::WARN);
-					}
-				}
+		if(is_null($valueFromLDAP) && $quotaAttribute !== '') {
+			$aQuota = $this->access->readAttribute($this->dn, $quotaAttribute);
+			if($aQuota && (count($aQuota) > 0) && $this->verifyQuotaValue($aQuota[0])) {
+				$quota = $aQuota[0];
+			} else if(is_array($aQuota) && isset($aQuota[0])) {
+				$this->log->log('no suitable LDAP quota found for user ' . $this->uid . ': [' . $aQuota[0] . ']', \OCP\Util::DEBUG);
```
Comment (nickvergessen) [inline apps/user_ldap/lib/User/User.php]: log->debug() instead of constant usage?

File: apps/user_ldap/lib/User/User.php [hunk 2]
```diff
@@ -498,44 +498,39 @@ public function updateQuota($valueFromLDAP = null) {
...
 			}
+		} else if ($this->verifyQuotaValue($valueFromLDAP)) {
+			$quota = $valueFromLDAP;
 		} else {
-			if ($this->verifyQuotaValue($valueFromLDAP)) {
-				$quota = $valueFromLDAP;
-			} else {
-				$this->log->log('not suitable LDAP quota found for user ' . $this->uid . ': [' . $valueFromLDAP . ']', \OCP\Util::WARN);
-			}
+			$this->log->log('no suitable LDAP quota found for user ' . $this->uid . ': [' . $valueFromLDAP . ']', \OCP\Util::DEBUG);
```
Comment (nickvergessen) [inline apps/user_ldap/lib/User/User.php]: same here and everywhere :)

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]
Okay, its the backport already

File: apps/user_ldap/lib/User/User.php (hunk 1 above)
Comment (blizzz) [reply apps/user_ldap/lib/User/User.php]: Kept it as it was for backporting reasons

**Comment by blizzz** [comment]
failing test unrelated (ocassionally occuring files acceptance) → merging

**Pull request merged**